### PR TITLE
Item resource creator role to create item

### PR DESCRIPTION
**Pull request by chinnu-focaloid**
Related to https://oat-sa.atlassian.net/browse/ADF-212
Give permission to create Item for Item Resource creator role
Add Item content creator permission for css authoring

### CODE REVIEW

**Review by boajer — APPROVED** [review]
Please do not forget to change depency to match https://github.com/oat-sa/extension-tao-item/pull/500/

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: manifest.php [hunk 1]
```diff
@@ -119,6 +119,16 @@
             TaoItemsRoles::ITEM_CONTENT_CREATOR,
             ['ext' => 'taoQtiItem', 'mod' => 'QtiCssAuthoring', 'act' => 'load']
         ],
+        [
+            'grant',
```
Comment (gabrielfs7) [inline manifest.php:123]: Lets use constants

Comment (chinnu-focaloid) [reply manifest.php]: Done

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by gabrielfs7 — APPROVED** [review]

**Comment by Natallia-Radkevich** [comment]
QA verified on  http://test-chinnufrancis.playground.kitchen.it.taocloud.org:48831/tao/Main/login

**Pull request merged**